include/odp/api/spec/classification.h

```diff
@@ -115,6 +115,10 @@ typedef union odp_cls_pmr_terms_t {
 		uint64_t	sctp_sport:1;
 		/** Destination SCTP port */
 		uint64_t	sctp_dport:1;
+		/** GTPU Tunnel endpoint identifier, Matches TEID filed in GTPv1 header.
+ 		 * i.e, Matches GTPU TEID value only if IP protocol is UDP and
+		 * UDP destination port is 2152.*/
+		uint64_t	gtpu_teid:1;
```

Comment:
I am still not sure if the description i adequate. Do GTPv2 packets match? If they do, do they match if the TEID-present flag is not set?

There is a typo (filed -> field) and capitalization issues (neither of "matches" should be capitalized, but i.e. should if it starts a sentence).

Reply:
GTPv2 won't match. This is only for GTPv1. Will fix typo.

Reply:
I think the description is still not totally clear. The title mentiones GTP-U (but does this not match GTP-C too (and GTP')?) and version 1, but second sentence talks only about UDP port number.

How about this in odp_cls_pmr_terms_t:

		/** GTPv1 tunnel endpoint identifier */
		uint64_t gtp_teid:1;

and this in odp_cls_pmr_term_t:

		/** GTPv1 tunnel endpoint identifier (val_sz = 4)
		  *
		  * Matches if and only if IP protocol is UDP, UDP destination port
		  * is 2152 and the UDP payload interpreted as GTP header has GTP
		  * version 3 and TEID as specified. 
		  */
		ODP_PMR_GTP_TEID,

Does the above match the intended semantics? Since GTP version is checked, it would
be good to have the validation test check that GTPv2 does not match.

Reply:
"GTP header has GTP version 3" you mean "GTP header has GTP version 1" right? not version 3.

Reply:
Yes, I meant 1, not 3.